test/common/ws/serverMaster.js

```diff
@@ -73,16 +74,17 @@ WSServerMaster.generatePeerHeaders = function (headers) {
 	}
 
 	var operatingSystems = ['win32', 'win64', 'ubuntu', 'debian', 'centos'];
+	var httpPort = random.number(1024, 65535);
```

Comment:
@robladbrook Second param for `random.number` is exclusive range. Should we not use `65536` to make sure we can use `65535` as port?